vitest.config.js

```diff
@@ -0,0 +1,35 @@
+import { fileURLToPath } from 'node:url';
+import { mergeConfig } from 'vite';
+import { configDefaults, defineConfig } from 'vitest/config';
+import viteConfig from './vite.config';
+
+const isCI = process.env.CI === 'true';
+
+export default mergeConfig(
+  viteConfig,
+  defineConfig({
+    test: {
+      globals: true,
+      environment: 'happy-dom',
+      exclude: [...configDefaults.exclude],
+      root: fileURLToPath(new URL('./', import.meta.url)),
+      watch: false,
+      setupFiles: ['./vitest.setup.js'],
+      // coverage: {
```

Comment:
Is the desired behavior to run our tests without coverage? Or are these configuration options just temporarily disabled?

Reply:
Eventually, we'll definitely want to enable this but since there aren't any tests at the moment nor anywhere to publish the test coverage, I figured I'd leave it here commented out. 

Once we have some tests in place, we should adjust the CI pipeline to execute the tests and coverage measurement. For the latter, we can publish the stats in the PRs as a comment and ideally publish those to a tool like CodeClimate (or similar).